src/routes/welcome/container/index.tsx

```diff
@@ -1,16 +1,12 @@
 import * as React from 'react'
-import { connect } from 'react-redux'
-
-import Layout from '../components/Layout'
-
-import selector from './selector'
+import { WelcomeLayout } from 'src/routes/welcome/components/index'
 
 import Page from 'src/components/layout/Page'
 
-const Welcome = ({ provider }) => (
+const Welcome = (): React.ReactElement => (
   <Page align="center">
-    <Layout provider={provider} />
+    <WelcomeLayout />
   </Page>
 )
 
-export default connect(selector)(Welcome)
+export default Welcome
```

Comment:
based on your latest changes, getting rid of the `default` export would be nice here too.

Reply:
Yep I agree but I think it's not possible because we are loading welcome as a lazy component and we need default export in that case, as far as I now